src/main/java/org/broadinstitute/hellbender/engine/spark/SparkContextFactory.java

```diff
@@ -51,6 +53,13 @@
             .put("spark.ui.enabled", Boolean.toString(SPARK_DEBUG_ENABLED))
             .put("spark.kryoserializer.buffer.max", "256m")
             .put("spark.hadoop.fs.file.impl.disable.cache", "true") // so NonChecksumLocalFileSystem is not cached between tests
+            // configure the gcs-connector
+            .put("spark.hadoop.fs.gs.impl", "com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem")
```

Comment:
We're going to have to add these to the spark defaults in gatk-launch as well I think in order to get this to work outside of tests.  Lets do that as a separate PR though since it will require some changes to gatk-launch. #3237